src/css/styles.css

```diff
@@ -2,6 +2,124 @@
 @import './global.css';
 
 body {
-  font-family: 'Nunito', sans-serif; /* Using Nunito font family*/
-  background-color: var(--primary-color); /*using a css variable*/
+  background-color: var(--light-blue-color);
+}
+
+.meals {
+  max-width: 774px;
+  margin: auto;
+}
+
+.title {
+  font-family: 'Permanent Marker', cursive;
+  height: 35px;
+  font-size: 24px;
+  color: var (--dark-gray-color);
+  font-weight: 400;
+  line-height: 35px;
+  margin-left: 10px;
+}
+
+.meals-card {
+  display: grid;
+  grid-template: repeat(2, 50%) / repeat(3, 33.3%);
+  align-items: center;
+  }
+
+.meals-content {
+  background-color: var(--white-color);
+  margin: 10px;
+  padding: 10px;
+  border-radius: 5px;
+  box-shadow: 0 5px 5px var(--light-gray-color);
+}
+
+.meals-content p {
+  font-family: 'Permanent Marker', cursive;
+  font-size: 14px;
+  color: var(--black-color);
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  text-align: center;
+}
+
+figure {
```

Comment:
nit: we could move this next to the body style (just to keep all the tags at the beginning)

Reply:
ohh ok. What do you think about footer, can it stay right where it is?

Reply:
ohh good question, i think it's ok to keep it where it is to have all the footer styles together